### PR TITLE
edit comment in SingleShot.java

### PR DESCRIPTION
**Pull request by phm0127**
edit comment in SingleShot.java

Signed-off-by: Park Hyeoung-Min <phm0127@gmail.com>

### CODE REVIEW

**Comment by myungjoo** [comment]
In the commit title and its contents, please declare what's the purpose and short description of the "edit".

Simply saying "edit comment in SingleShot.java" cannot justify the commit.

**Comment by myungjoo** [comment]
Sign-off in commit message is missing. You have sign-off at PR message, but the commit does not have it.
You can add it by
```
$ git commit --amend -s
```
if you have configured git with your name and email address.

Then, push with --force option to update the PR.

**Comment by myungjoo** [comment]
FYI: https://github.com/nnstreamer/nnstreamer/issues/2519

File: api/android/api/src/main/java/org/nnsuite/nnstreamer/SingleShot.java [hunk 1]
```diff
@@ -51,7 +51,7 @@
      * @param model The path to the neural network model file
      *
      * @throws IllegalArgumentException if given param is invalid
-     * @throws IllegalStateException if failed to construct the pipeline
+     * @throws IllegalStateException if this failed to construct the pipeline
```
Comment (jaeyun-jung) [inline api/android/api/src/main/java/org/nnsuite/nnstreamer/SingleShot.java:54]: please update the description all constructors in SingleShot class.

**Comment by myungjoo** [comment]
1. Can you please use realname for Sign-off?
2. Can you use email addresses consistently?

**Comment by myungjoo** [comment]
DCO (official sign-off checker of Linux Foundation) complains:

---

You only have one commit incorrectly signed off! To fix, head to your local branch and run:
```
$ git commit --amend --signoff
```
Now your commits will have your sign off. Next run
```
git push --force-with-lease origin master
```
Commit sha: 64d4102, Author: 박형민 (Park Hyeoung Min), Committer: 1000min; Expected "박형민 (Park Hyeoung Min) o_ogog@naver.com", but got "Park Hyeoung-Min phm0127@gmail.com".

**Comment by myungjoo** [comment]
Not a critical code and the DCO error appears to be false positive. (He appears to use his real name here).
I'm ignoring DCO and merging this.

**Pull request merged**